interface/usergroup/addrbook_edit.php

```diff
@@ -465,7 +465,7 @@ function typeSelect(a) {
   <td>
    <input type='entry' size='10' name='form_city' maxlength='30'
     value='<?php echo attr($row['city']); ?>' class='inputtext' />&nbsp;&nbsp;&nbsp;
-   <b><?php echo xlt('State')."/".xlt('county'); ?>:</b>&nbsp;&nbsp;&nbsp; <input type='entry' size='10' name='form_state' maxlength='30'
+   <b><?php echo xlt('State')."/".xlt('country'); ?>:</b>&nbsp;&nbsp;&nbsp; <input type='entry' size='10' name='form_state' maxlength='30'
```

Comment:
hey @HenriettaWilla  why did you correct county? I think State/county was correct considering we are using American Grammar here. Check this out [https://en.wikipedia.org/wiki/County_(United_States)](url). @KoniKodes  @muarachmann who do you think?

Reply:
I am sorry about that I thought it was a typo error